tests/util/random_numbers.py

```diff
@@ -0,0 +1,68 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from random import randrange, uniform
+
+from octobot_commons.enums import PriceIndexes
+
+MAX_PRICE = 10e7
```

Comment:
would it be possible to also add a min price ? something like 10e-7

Reply:
The current min is 0,why do you want to increase it?

Reply:
I realize my comment was not very clear, is it possible to have a range going to a 7 digits representation ? some price data have this type of precision

Reply:
As you can see [here](https://pynative.com/python-get-random-float-numbers) the larger the number the smaller the precision and the smaller the number the greater the precision (up to the size of a python float). I don't think we need to add something to the current implementation to have a big precision. The only thing we can do is reduce the max in some cases to ensure having a lots of digits.

Reply:
perfect